packages/eslint-config-universe/shared/prettier.js

```diff
@@ -1,7 +1,12 @@
 module.exports = {
-  extends: ['prettier', 'prettier/@typescript-eslint', 'prettier/babel', 'prettier/react'],
+  extends: ['prettier'],
   plugins: ['prettier'],
   rules: {
-    'prettier/prettier': ['warn'],
+    'prettier/prettier': [
+      'warn',
+      {
+        arrowParens: 'avoid',
+      },
```

Comment:
If you want to remove this setting I can remove this even now, but it would result in ~300 format warnings in the workspace, my goal was to mimic the current setup as much as possible to reduce the number of format changes after migration.

Reply:
yea another commit

Reply:
I'm not sure if I like the enforcement of parenthesis everywhere, what do you think about that formatting rule after looking at the changes?

Maybe this Prettier option should be retained and placed within other Prettier config entries in `.prettierrc` (https://github.com/expo/expo/blob/master/packages/eslint-config-universe/.prettierrc)?

Reply:
We use the parentheses in our other codebases and plan to do so here but because of how much code it changes, a separate commit (and therefore separate PR) will be easier to read.

Reply:
To reduce the PR changeset size I have decided to revert PR to the state before `arrowParens` removal. 

The parenthesis reformat and new config option removal will be posted in the separate PR, if I will be able to fix the issues in this PR and got it merged.